README.md

```diff
@@ -5,91 +5,93 @@
 JWT utils for Fastify, internally uses [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken).
 
 ## Install
+
 ```
 npm i fastify-jwt --save
 ```
 
 ## Usage
+
 Register as a plugin. This will decorate your `fastify` instance with the standard [jsonwebtoken](https://github.com/auth0/node-jsonwebtoken) methods `decode`, `sign`, and `verify`; refer to their documentation to find how to use the utilities. It will also register `request.jwtVerify` and `reply.jwtSign`. You must pass a `secret` when registering the plugin.
 
 ```js
-const fastify = require('fastify')
+const fastify = require('fastify');
 fastify.register(require('fastify-jwt'), {
   secret: 'supersecret'
-})
+});
 
 fastify.post('/signup', (req, reply) => {
   // some code
-  const token = fastify.jwt.sign({ payload })
-  reply.send({ token })
-})
+  const token = fastify.jwt.sign({ payload });
+  reply.send({ token });
+});
 
 fastify.listen(3000, err => {
-  if (err) throw err
-})
+  if (err) throw err;
+});
 ```
 
 For verifying & accessing the decoded token inside your services, you can use a global `onRequest` hook to define the verification process like so:
 
 ```js
-const fastify = require('fastify')
+const fastify = require('fastify');
 fastify.register(require('fastify-jwt'), {
   secret: 'supersecret'
-})
+});
 
-fastify.addHook("onRequest", async (request, reply) => {
+fastify.addHook('onRequest', async (request, reply) => {
   try {
-    await request.jwtVerify()
+    await request.jwtVerify();
```

Comment:
why all these changes in the readme? 😲

Reply:
💵 says Prettier was used to to reformat everything on save.

@kamikazechaser please revert the linting changes and do not use Prettier when working on this project.

Reply:
My bad! My prettier runs globally. Should have checked before pushing. Resolved.